subsys/bluetooth/controller/hci_internal.c

```diff
@@ -762,6 +762,14 @@ static uint8_t vs_cmd_put(uint8_t const * const cmd,
 #endif /* CONFIG_BT_CTLR_TX_PWR_DYNAMIC_CONTROL */
 #endif /* CONFIG_BT_HCI_VS_EXT */
 
+#if defined(CONFIG_SDC_READ_CHIP_TEMP)
```

Comment:
Now that we make the inclusion of the zephyr vs hci commands configurable, we should not call `sdc_hci_cmd_vs_zephyr_read_supported_commands()`. We should rather implement it here like call we have done for `supported_commands()`

Reply:
I don't understand this comment. Do you mean not have read_chip_temperature = 1; in the SDC implementation of sdc_hci_cmd_vs_zephyr_read_supported_commands() or something else ?

Reply:
We need to use an implementation of "Zephyr Read Supported commands" which can set `read_chip_temperature` to either 0 or 1 depending on CONFIG_SDC_READ_CHIP_TEMP. For "HCI Read supported commands" we have created our own implementation of that function instead of calling the SDC API.

Reply:
OOB: will be handled in a separate PR